packages/ses/src/error/stringify-utils.js

```diff
@@ -120,7 +125,28 @@ const bestEffortStringify = (payload, spaces = undefined) => {
           // and their remote presences in the first place.
           return `[${val[toStringTagSymbol]}]`;
         }
-        return val;
+        if (isArray(val)) {
+          return val;
+        }
+        const names = keys(val);
+        if (names.length < 2) {
+          return val;
+        }
+        let sorted = true;
+        for (let i = 1; i < names.length; i += 1) {
+          if (names[i - 1] >= names[i]) {
+            sorted = false;
+            break;
+          }
+        }
+        if (sorted) {
+          return val;
+        }
+        arraySort(names);
+        const entries = arrayMap(names, name => [name, val[name]]);
+        const replacement = fromEntries(entries);
+        setAdd(seenSet, replacement);
```

Comment:
This extra registration is pointless. Since the `replacement` is always fresh, it will never be found in the `seenSet`

Reply:
Fixed